src/scitacean/ontology/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""Tools for working with ontologies."""
+
+import bz2
+import importlib.resources
+import json
+import re
+from functools import cache
+
+from ..model import Technique
+
+
+def _load_ontology(name: str) -> object:
+    """Load an ontology from the package resources.
+
+    Note that the ontology file was generated using a script in
+    ``tools/ontologies`` in the Scitacean repository.
+    """
+    with (
+        importlib.resources.files("scitacean.ontology")
+        .joinpath(f"{name}.json.bz2")
+        .open("rb") as raw_f
+    ):
+        with bz2.open(raw_f, "rb") as f:
+            return json.loads(f.read())
+
+
+@cache
+def expands_techniques() -> dict[str, list[str]]:
+    """Load the ExPaNDS experimental techniques ontology.
+
+    Returns
+    -------
+    :
+        A dict mapping from technique ids (IRIs) to labels.
+        The first element of the list is the primary label.
+        All labels are lowercase and contain no leading or trailing whitespace.
+    """
+    return _load_ontology("expands_techniques")  # type: ignore[return-value]
```

Comment:
If this function is for notebooks, maybe we can wrap it in html repr, otherwise ignore me.